### PR TITLE
Release tmt-1.29.0

### PR DESCRIPTION
**Pull request by lukaszachy**
Follow the steps below to create a new major or minor release:

- [x] Run the full test coverage using ``tmt -c how=full run``
- [x] Use ``git log --oneline --no-decorate x.y-1..`` to generate the changelog
- [x] Update ``README`` with new contributors since the last release
- [x] Add a ``Release tmt-x.y.0`` commit with the specfile update
- [ ] Create a pull request with the commit, ensure tests pass, merge it

Release a new package to Fedora and EPEL repositories:

- [ ] Move the ``fedora`` branch to point to the new release
- [ ] Tag the commit with ``x.y.0``, push tags ``git push --tags``
- [ ] Create a new `github release`__ based on the tag above
- [ ] Check Fedora `pull requests`__, make sure tests pass and merge

Finally, if everything went well:

- [ ] Close the corresponding release milestone
- [ ] Once the `copr build`__ is completed, move the ``quay`` branch
  to point to the release commit as well to build fresh container
  images.

Handle manually what did not went well:

* If the automation triggered by publishing the new github release
  was not successful, publish the fresh code to the `pypi`__
  repository manually using ``make wheel && make upload``
* If there was a problem with creating Fedora pull requests, you
  can trigger them manually using ``/packit propose-downstream``
  in any open issue.
* If running `packit propose-downstream`__ from your laptop make
  sure that the ``post-upstream-clone`` action is disabled in
  ``.packit.yaml`` to prevent bumping the devel version.

### CODE REVIEW

File: tmt.spec [hunk 1]
```diff
@@ -147,6 +147,50 @@ install -pm 644 %{name}/steps/provision/mrack/mrack* %{buildroot}/etc/%{name}/
 %config(noreplace) %{_sysconfdir}/%{name}/mrack*
 
 %changelog
+* Mon Nov 06 2023 Lukáš Zachar <lzachar@redhat.com> - 1.29.0
+- Refactor generating of stories and lint check docs
```
Comment (psss) [inline tmt.spec:151]: Last commit from `main` seems to be missing:

    Update and polish hardware requirement docs

**Review by psss — CHANGES_REQUESTED** [review]
Looks good, just one missing commit.

**Comment by lukaszachy** [comment]
@psss Attempt in 1.30 release notes, improvements appreciated.

**Comment by psss** [comment]
> @psss Attempt in 1.30 release notes, improvements appreciated.

Done in 0e66e267. Please, have a look if everything's ok.

**Comment by lukaszachy** [comment]
@psss Looks good. thanks

**Review by psss — APPROVED** [review]

**Pull request merged**